### DISCUSSION TITLE
Mac and Linux use

### DISCUSSION THREAD

**Discussion by highfestiva** [highfestiva/finplot#409] [Q&A]
I only run finplot under Windows, and have no idea if it's working properly, or even working at all, under Mac or Linux. So would love to hear from any users with experience from those platforms. Thanks!

**Comment by atilagf1** [comment]
I switch from Windows to Fedora linux with KDE, it is working well. However I tried to take a screenshot, but it just show a white image.

**Reply by highfestiva** [reply]
Sorry to hear the screenshots aren't working, but great that it seems to function ok overall. Did you have your window on your primary screen? If there's anybody here who has some understanding of primary screen/grabWindow/save png in Qt, and knows what might be causing this under Linux, feel free to pitch in! :)

**Comment by quest2nest** [comment]
I'm using on Debian stable (Linux). Works perfectly well. Screenshots are working. I can hook into the underlying pyqt libraries and stuff.

Amazing piece of software!

**Reply by highfestiva** [reply]
Great, glad to hear it!

**Comment by mukai90** [comment]
on ubuntu ,    
use(python)
   os.putenv("QT_QPA_PLATFORM","xcb")
and 
   sudo install libxcb-cursor0
